hbase-http/src/main/java/org/apache/hadoop/hbase/http/prometheus/PrometheusHadoopServlet.java

```diff
@@ -57,13 +60,19 @@ static String toPrometheusName(String metricRecordName, String metricName) {
    */
   @RestrictedApi(explanation = "Should only be called in tests or self", link = "",
       allowedOnPath = ".*/src/test/.*|.*/PrometheusHadoopServlet\\.java")
-  void writeMetrics(Writer writer) throws IOException {
+  void writeMetrics(Writer writer, boolean desc) throws IOException {
```

Comment:
Maybe 'describe' is better argument name than 'desc'.

Reply:
I use 'desc', because 'description' is used as a local variable name in the function